app/models/user.rb

```diff
@@ -488,7 +488,7 @@ def needs_feed_update?
   end
 
   def validate_email_dns?
-    email_changed? && !external? && !(Rails.env.test? || Rails.env.development?)
+    email_changed? && !external? && !Rails.env.local? # rubocop:disable Rails/UnknownEnv
```

Comment:
This rubocop disable is to prevent what is effectively a false positive - https://github.com/rubocop/rubocop-rails/pull/894 - we can remove the disable after that is merged in.

Reply:
This has been merged, but maybe not released, or we need to upgrade `rubocop-rails`?

Reply:
We could merge this now and the disable comment here will stop rubocop from complaining. Once rubocop-rails is released with this change and we update, we can remove the disable.

Alternately, in the event its released and we update before this PR is merged, I can rebase this and remove the disable.

Reply:
Maybe do the `--regen-todo` to add it to the TODO file, then it should go away next time, and you don't need to touch these files after the new release